apps/spruce/src/pages/task/taskTabs/index.tsx

```diff
@@ -69,18 +75,16 @@ export const TaskTabs: React.FC<TaskTabProps> = ({ isDisplayTask, task }) => {
         key="task-tests-tab"
         data-cy="task-tests-tab"
         name={
-          <span>
-            {failedTestCount ? (
-              <TabLabelWithBadge
-                badgeText={failedTestCount}
-                badgeVariant="red"
-                dataCyBadge="tests-tab-badge"
-                tabLabel="Tests"
-              />
-            ) : (
-              "Tests"
-            )}
-          </span>
+          failedTestCount ? (
+            <TabLabelWithBadge
+              badgeText={failedTestCount}
+              badgeVariant="red"
```

Comment:
nit. can we `import Variant as BadgeVariant` and use the values on these components?